shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Options",
+    "description": "Workflow options",
+    "type": "object",
+    "properties": {
+        "status": {
+            "description": "The current status of a workflow execution",
+            "$ref": "workflow-status-schema.json"
```

Comment:
Do we need workflow-specific versions as it used to be for the fields bellow?
I think we can make this structure generic-enough and mandatory.

Reply:
workflow versioning was taken as is from Parodos. I think they do not have value in current implementation. I am ok to remove it.

Reply:
If we remove it, we can move the status "one level up" - the structure of `workflow-status-schema.json` will be content of this object

Reply:
moved one level up